sdk/core/azure-core-xml/cgmanifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://json.schemastore.org/component-detection-manifest.json",
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
```

Comment:
I don't see libxml on this list of dependencies, does it come from some other element?

Reply:
done